ideas/90-branch-perf-stats.md

```diff
@@ -0,0 +1,105 @@
+---
+id: 090-branch-perf-stats
+title: Make branch performance stats visible
+status: draft
+created: 2018-03-12
+category: core
+lead-contributor: PombeirP
+contributors:
+    -
+    -
+    -
+exit-criteria: no
+success-metrics: no
+clear-roles: no
+future-iteration: no
+roles-needed:
+    - DevOps
+    - Go dev
+okrs:
+   - "[P4]: Objective: Implement continuous delivery"
+   - "[P0]: KR: >80% automated test coverage"
+---
+
+## Preamble
+
+    Idea: 090
+    Title: Make branch performance stats visible
+    Status: Draft
+    Created: 2018-03-12
+
+## Summary
+
+Right now there is no visibility of how a PR impacts performance before it is merged to develop. It would be highly beneficial to have automated performance tests run and attach info to each Jenkins build, so that we can have historical records and compare current branch with parent branch.
+
+## Swarm Participants
+
+- Lead Contributor: PombeirP
+- Contributor (Go): TBD
+- Contributor (DevOps): TBD
+
+## Product Overview
+
+Right now, we run end-to-end tests but we don’t collect performance metrics on them. That’s throwing away a lot of valuable information that can inform discussions and decisions down the road. It’s important to have metrics on every PR branch and not just on develop, so that contributors have visibility into how their changes will potentially be affecting the stable branch.
```

Comment:
This idea seems a bit related to https://github.com/status-im/ideas/issues/91 which is a P0, maybe it makes sense to attack that one before this one? (I know this is a draft, but just thinking out loud)